README.md

```diff
@@ -1 +1,42 @@
-# tekton-watcher
+# Tekton Watcher
+
+Tekton Watcher is a service to react to events that occur in Tekton Pipelines and Tasks.
```

Comment:
I wonder if we should to use the terms pipeline runs and task runs since for Tekton pipelines and tasks are merely blueprints.

Reply:
I thought about that too, my concern is to confuse the user with the specific verbiage